index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Wbpack Exercise</title>
+</head>
+<body>

```

Comment:
Great work so far! I you html file would look better if you created an article or section container tag to separate the various sections of the page to keep them organized and easy to read. :)